git.mk

```diff
@@ -37,10 +37,31 @@ git: git-setup openssl curl pcre2 gettext libidn2
 			execve(SHELL_PATH, (char *const *) out.argv, \
  			       (char *const *) childenv); \
 		}' $(BUILD_WORK)/git/run-command.c
+	+$(MAKE) -C $(BUILD_WORK)/git all \
+		uname_S=Darwin \
+		HOST_CPU=$(GNU_HOST_TRIPLE) \
+		DESTDIR=$(BUILD_STAGE)/git \
+		MANDIR=/usr/share/man \
+		NO_DARWIN_PORTS=1 \
+		NO_FINK=1 \
+		NO_APPLE_COMMON_CRYPTO=1 \
+		INSTALL_SYMLINKS=1 \
+		NO_INSTALL_HARDLINKS=1
+	+$(MAKE) -C $(BUILD_WORK)/git/Documentation man install \
+		uname_S=Darwin \
```

Comment:
Could you combine these make args into one variable on up top called $(GIT_MAKEFLAGS) or something?